nixos/doc/manual/man-nixos-option.xml

```diff
@@ -119,4 +119,8 @@ Defined by:
    bug, please report to Nicolas Pierron.
   </para>
  </refsection>
+ <refsection>
+   <title>See also</title>
+   <para>configuration.nix (5)</para>
```

Comment:
IIRC docbook has its own syntax for referencing man pages: https://tdg.docbook.org/tdg/4.5/manvolnum.html

Reply:
Thanks!